### ISSUE TITLE
Delete associated docs when reverting release

### ISSUE THREAD

**Issue by ericmj** [hexpm/hex_web#85]

**Comment by jcspencer** [comment]
Is this as simple as adding a the following block [here](https://github.com/hexpm/hex_web/blob/master/lib/hex_web/api/router.ex#L228)?

```elixir
paths = store.list_docs_pages(Path.join(name, version))
store.delete_docs("#{name}-#{version}.tar.gz")

Enum.each(paths, fn path ->
  store.delete_docs_page(path)
end)
```

If this is corrrect, I'll open a PR

**Comment by ericmj** [comment]
Yeah, I think that should be enough.

**Comment by jcspencer** [comment]
@ericmj Should be fixed in #96
